frontend/src/components/InfoLink.tsx

```diff
@@ -0,0 +1,27 @@
+import {Link} from '@cloudscape-design/components'
+import {ReactElement} from 'react'
+import {useTranslation} from 'react-i18next'
+import {useHelpPanel} from './help-panel/HelpPanel'
+
+interface InfoLinkProps {
```

Comment:
nitpick(non-blocking): more of a Typescript thing, in this case you can use `type` instead of `interface` cause it's way more semantic.
```typescript
type InfoLinkProps = {
  ariaLabel: string
  helpPanel: ReactElement
}
```